python/paddle/profiler/profiler.py

```diff
@@ -316,14 +319,78 @@ class Profiler:
                 p.stop()
                 p.summary()
 
+        4. Use profiler to get throughput and cost of the model
+
+            .. code-block:: python
+                :name: code-example-timer1
+
+                import paddle
+                import paddle.profiler as profiler
+                import numpy as np
```

Comment:
Please use Paddle's API create Tensor instead of numpy

Reply:
done.